src/ui/state/vaults.rs

```diff
@@ -106,19 +122,24 @@ impl State for HistoryState {
             ctx,
             self.warning.as_ref().into(),
             self.vaults.iter_mut().map(|v| v.view(ctx)).collect(),
+            self.vault_status_filter,
+            self.loading,
         )
     }
 
     fn load(&self) -> Command<Message> {
         Command::batch(vec![
             Command::perform(get_blockheight(self.revaultd.clone()), Message::BlockHeight),
-            Command::perform(list_vaults(self.revaultd.clone(), None), Message::Vaults),
+            Command::perform(
+                list_vaults(self.revaultd.clone(), Some(&VaultStatus::CURRENT)),
```

Comment:
small nit: maybe putting here
```
list_vaults(self.revaultd.clone(), Some(&self.vault_status_filter)),
```
So if we ever change the default vault status filter we won't introduce bugs? It could also prevent a strange state where the user changes the filter before the vault is loaded (should be impossible but who knows)

Reply:
good catch